Stripe/STPSectionHeaderView.h

```diff
@@ -0,0 +1,18 @@
+//
+//  STPSectionHeaderView.h
+//  Stripe
+//
+//  Created by Ben Guo on 1/3/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "STPTheme.h"
+
+@interface STPSectionHeaderView : UIView
```

Comment:
hm, is there any benefit to using `UITableViewHeaderFooterView` in this case? Seems like its main feature is reuse, which we wouldn't be using